source/scss/_pf-basic-variables.scss

```diff
@@ -456,3 +460,33 @@ $tag-font-weight:           $pf-font-weight-bold;
 // Use a higher than normal value to ensure completely rounded edges when
 // customizing padding or font-size on labels.
 // $tag-pill-border-radius:    10rem !default;
+
+
+// Form states and alerts
+//
+// Define colors for form feedback states and, by default, alerts.
```

Comment:
Should icon colors also be included here so that all the colors are defined in one place?